src/solve.jl

```diff
@@ -32,7 +32,7 @@ function init{algType<:OrdinaryDiffEqAlgorithm,recompile_flag}(
   reltol=nothing,
   qmax=qmax_default(alg),qmin=qmin_default(alg),
   qsteady_min = qsteady_min_default(alg),
-  qsteady_max = qsteady_min_default(alg),
```

Comment:
oh shit, no wonder I never noticed a difference when using it... I wonder if this line effects other methods / benchmarks.